spec/system/posts_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
```

Comment:
<img width="752" alt="Screen Shot 2022-09-27 at 5 15 52 PM" src="https://user-images.githubusercontent.com/83717285/192647793-fdd986b1-ca20-4e7e-9ec9-e2ecca8a7586.png">

- [ ] You have done an excellent job and your tests are written properly, congrats on it! 🎉  However, they are not passing but I guess it's probably because you have some different users or posts in your database, to avoid this you could try destroying the stored data in the `before do` callback and then creating the subject user and post.

Reply:
Hi @smunozmo, thank you for the review, but I don't use create records in the test process, I use db seeds instead.
So if you try running `rails db:seed RAILS_ENV=test` you will have all the necessary data for testing.

Reply:
@teckim that's perfect! I would suggest adding that command to the testing instructions of your README file, this way the next reviewer will be aware of it. 🚀